php/libraries/NDB_Page.class.inc

```diff
@@ -409,10 +425,16 @@ class NDB_Page implements RequestHandlerInterface
      * @param string $format  See QuickForm documentation.
      *
      * @return void
+     *
+     * @note The fourth parameter ($format) was removed because
+     *       LorisForm::addRule does not have this fourth parameter. It is
+     *       still included in the function declaration because this function
+     *       may be called with four parameters elsewhere in the codebase!
```

Comment:
As I explain in the note, I removed the "format" from the actual LorisForm::addRule method call, but I kept it in the method definition because when I tried to remove it, I got errors because this method is called with four parameters  in some of the modules. Just like for the addHeader method, I would have to change all the times the method is called in the backend in order to remove the format parameter!